ros/src/computing/perception/detection/lidar_detector/packages/lidar_point_pillars/launch/lidar_point_pillars.launch

```diff
@@ -0,0 +1,24 @@
+<launch>
+  <arg name="input_topic" default="/points_raw" />
+  <arg name="baselink_support" default="true" />
+  <arg name="reproduce_result_mode" default="true" />
+  <arg name="score_threshold" default="0.5" />
+  <arg name="nms_overlap_threshold" default="0.5" />
+  <arg name="pfe_onnx_file" default="" />
+  <arg name="rpn_onnx_file" default="" />
```

Comment:
remove the default value of `pfe_onnx_file` and `rpn_onnx_file`, these are required.